docs/README.md

```diff
@@ -157,7 +157,7 @@
 * [Extending the Monitoring Stack](development/monitoring-stack.md)
 * [How to create log parser for container into fluent-bit](development/log_parsers.md)
 * [`PriorityClasses` in Gardener Clusters](development/priority-classes.md)
-* [High Availability Of Deployed Components](development/high-availability.md)
+* [High Availability Of Deployed Components](development/high-availability-dev.md)
```

Comment:
Cosmetic suggestion:

```suggestion
* [High Availability Of Deployed Components](development/high-availability-of-components.md)
```

to make to closer to the document title.

Reply:
I agree with that suggestion. @Kostov6 What do you think?